### PR TITLE
cleaned up plots

### PR DESCRIPTION
**Pull request by danielecertini90**

### CODE REVIEW

File: plot.py [hunk 1]
```diff
@@ -3,9 +3,13 @@
 import matplotlib.pyplot as plt
 from matplotlib.colors import LogNorm
 import seaborn as sb
+plt.rcParams['font.size'] = 24
+plt.rcParams['font.family'] = 'Times New Roman'
 
 # Read in the file
-df = pd.read_csv('seed/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
+# df = pd.read_csv('seed/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
```
Comment (mahditaani) [inline plot.py]: This line is unnecessary. Remove it

File: plot.py [hunk 2]
```diff
@@ -3,9 +3,13 @@
...
+df = pd.read_csv('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraWing/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
+#df = pd.read_csv('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraSeed/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
```
Comment (mahditaani) [inline plot.py]: Please remove

File: plot.py [hunk 3]
```diff
@@ -3,9 +3,13 @@
 import matplotlib.pyplot as plt
 from matplotlib.colors import LogNorm
 import seaborn as sb
+plt.rcParams['font.size'] = 24
+plt.rcParams['font.family'] = 'Times New Roman'
 
 # Read in the file
-df = pd.read_csv('seed/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
+# df = pd.read_csv('seed/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
+df = pd.read_csv('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraWing/XYZ_S1.dat', sep='\t', skiprows=[0,1], names=['EM','EX','I'])
```
Comment (mahditaani) [inline plot.py]: Remove directory and filename out of command (for clarity)
e.g.
```dir = '/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraWing/'```
```filename = 'XYZ_S1.dat'
```pd.read_csv('dir+filename...```

File: plot.py [hunk 4]
```diff
@@ -65,24 +69,31 @@
     axs[1].set_xlabel('Wavelength of Incident Light [nm]')
     axs[1].set_ylabel('Wavelength of Reflected Light [nm]')
 '''
-fig1, axs1 = plt.subplots(1)
+fig1, axs1 = plt.subplots(1,figsize=(10,10))
```
Comment (mahditaani) [inline plot.py]: Why only the first plot ?

File: plot.py [hunk 5]
```diff
@@ -65,24 +69,31 @@
...
 fig2, axs2 = plt.subplots(1)
 fig3, axs3 = plt.subplots(1)
 # Plot Slice
-axs1.plot(xVals, yVals)
-axs1.set_title('Intensity of Reflected Light (Incident = 850-860nm)')
+axs1.plot(xVals, yVals, 'k-')
+# axs1.set_title('Intensity of Reflected Light (Incident = 850-860nm)')
```
Comment (mahditaani) [inline plot.py]: Remove please

File: plot.py [hunk 6]
```diff
@@ -65,24 +69,31 @@
...
 axs1.set_xlabel('Wavelength of Reflected Light [nm]')
 axs1.set_ylabel('[Arb]')
+
+plt.savefig('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraSeed/XYZ_R1_seed_plots/IntensityOfReflectedLight.png', dpi=300, bbox_inches="tight", pad_inches=0, transparent=True)
```
Comment (mahditaani) [inline plot.py]: see above comments

File: plot.py [hunk 7]
```diff
@@ -65,24 +69,31 @@
...
+#plt.savefig('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraSeed/XYZ_R1_seed_plots/IntensityOfReflectedLight.png', dpi=300, bbox_inches="tight", pad_inches=0, transparent=True)
```
Comment (mahditaani) [inline plot.py]: see above

File: plot.py [hunk 8]
```diff
@@ -65,24 +69,31 @@
...
+#plt.savefig('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraSeed/XYZ_R1_seed_plots/IntensityOfReflectedLightvsIncidentLight1.png', dpi=300, bbox_inches="tight", pad_inches=0, transparent=True)
 # Plot 3D Normal Scale
-sb.heatmap( df, ax=axs3)
-axs3.set_title('Intensity of Reflected Light vs Incident Light')
+sb.heatmap( df, ax=axs3, cmap='viridis')
+# axs3.set_title('Intensity of Reflected Light vs Incident Light')
 axs3.set_xlabel('Wavelength of Incident Light [nm]')
 axs3.set_ylabel('Wavelength of Reflected Light [nm]')
 plt.tight_layout()
+#plt.savefig('/Users/danielecertini/Desktop/PhD_University_of_Edinburgh/Alsomitra_macrocarpa/SpectrometerKingsCollegeLondonMahdi/AlsomitraSeed/XYZ_R1_seed_plots/IntensityOfReflectedLightvsIncidentLight2.png', dpi=300, bbox_inches="tight", pad_inches=0, transparent=True)
```
Comment (mahditaani) [inline plot.py]: see above

**Review by mahditaani — CHANGES_REQUESTED** [review]
Good work on the changes! Just a few minor comments.

File: plot.py (hunk 4 above)
Comment (danielecertini90) [reply plot.py]: It was a test. Reverting

File: .gitignore [hunk 9]
```diff
@@ -1 +1,2 @@
 *desktop.ini
+.DS_Store
```
Comment (mahditaani) [inline .gitignore:2]: Great addition!

**Review by mahditaani — APPROVED** [review]
Nice one.

**Pull request merged**